server/storage/sql_models.go

```diff
@@ -11,6 +11,16 @@ const (
 	changeCategoryDeletion = "deletion"
 )
 
+// TUFFileTableName returns the name used for the tuf file table
+var TUFFileTableName = func() string {
```

Comment:
Agreed with @endophage , I don't see the explicit benefit we can from these functions, it's not that directly and clear.